app/source/main.c

```diff
@@ -172,6 +184,8 @@ int main()
 
   ACTA_GetAccountCount(&act_account_count);
 
+  u32 btn_selected = 0;
```

Comment:
With the current logic, clicking left or right will always set one of the accounts, but by default nothing is selected.

What are thoughts on:
- Adding an `Account` enum to make it easier to read where Nintendo vs. Pretendo logic is happening
- Typing `btn_selected` as `Account`
- Defaulting the selection to the currently active account, which can simplify logic later and might provide a more consistent UX

```cpp
enum Account {
  Nintendo = 1,
  Pretendo = 2,
}
```

In a future code update (not necessarily this PR), someone would then be able to reuse `Account` for `switchAccounts` and `current_account`.